README.md

```diff
@@ -19,3 +19,11 @@ Run a simple inverter design thru the [OpenLane](https://github.com/The-OpenROAD
 Draw a simple mosfet for the [open source SKY130 PDK](https://github.com/google/skywater-pdk/) using [MAGIC](https://github.com/RTimothyEdwards/magic) and simulate it in an inverter circuit using [PySpice](https://pyspice.fabrice-salvaire.fr/).
 
 ![mosfet layout](img/mosfet.png)
+
+### XLS adder with OpenLane [![colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/chipsalliance/silicon-notebooks/blob/main/xls-adder-openlane.ipynb)
+
+[xls-adder-openlane.ipynb](xls-adder-openlane.ipynb)
+
+Run a simple 1-bit adder design using the [XLS](https://google.github.io/xls/) high level synthesis toolkit thru the [OpenLane](https://github.com/The-OpenROAD-Project/OpenLane/) GDS to RTL flow targeting the [open source SKY130 PDK](https://github.com/google/skywater-pdk/).
+
+![adder layout](https://drive.google.com/uc?export=view&id=1DYKbQg2kJJxHAZ5afZgVZU3-ZxO6lQnA)
```

Comment:
can you add the image to the pull request instead?

Reply:
![adder_layout](https://user-images.githubusercontent.com/60669669/192484983-4205f5da-2cf6-4bae-8cba-624ae054df8b.jpg)

Reply:
sorry, what I meant is adding it as part of the `img/` subdirectory along side your pull request.